tests/test_compile_source.py

```diff
@@ -30,3 +31,17 @@ def foo() -> int128:
     return 42
     """
     vvm.compile_source(source, vyper_version=version_str)
+
+
+def test_compile_metadata(foo_source, vyper_version):
+    if vyper_version <= Version("0.3.1"):
+        raise pytest.skip("metadata output not supported in vyper < 0.3.2")
```

Comment:
what does `raise pytest.skip` do?

Reply:
based on conversation in https://github.com/vyperlang/vvm/pull/22, please change this to regular `pytest.skip`